python/docs/source/guide/timestreams.md

```diff
@@ -1,3 +1,54 @@
+---
+file_format: mystnb
+kernelspec:
+  name: python3
+  disply_name: Python 3
+mystnb:
+  execution_mode: cache
+---
+
 # Timestreams
 
-## Continuity
\ No newline at end of file
+Kaskada is built on the concept of a _Timestream_.
+Each Timestream is ordered by time and partitioned by _entity_.
+This makes it easy to focus on events happening over time and how aggregations change.
+
+```{code-cell}
+---
+tags: [remove-input]
+---
+import kaskada as kd
+kd.init_session()
+data = "\n".join(
+    [
+        "time,key,m",
+        "1996-12-19T16:39:57,A,5",
+        "1996-12-19T17:42:57,B,8",
+        "1996-12-20T16:39:59,A,17",
+        "1996-12-23T12:18:57,B,6",
+        "1996-12-23T16:40:01,A,12",
+    ]
+)
+multi_entity = kd.sources.CsvString(data, time_column="time", key_column="key")
+
+kd.plot.render(
+    kd.plot.Plot(multi_entity.col("m"))
+)
+```
+
+## Continuity
+
+It is useful to consider two kinds of timestreams -- _discrete_ and _continuous_.
+Like the example we already saw, a discrete timestream consists of values at specific points in time.
+On the other hand, a continuous timestream represents a value that continues until it changes.
+While a discrete timestream contains values at specific points in time, a continuous timestream changes at specific points in time.
```

Comment:
I'm sure this will be confusing for first-time users, but it sounds like a good explanation that we can link or elaborate further on.

Reply:
Agreed.